SimPEG/Examples/DC_PseudoSection_Simulation.py

```diff
@@ -0,0 +1,220 @@
+from SimPEG import *
+# import simpegDCIP as DC
+from SimPEG import DCIP as DC
```

Comment:
right now DCIP isn't on the Examples or master branch, so this import is failing, we will need to get DCIP into master first